bids_prov/fsl_config.py

```diff
@@ -0,0 +1,60 @@
+from typing import Mapping, Union
+
+from boutiques.searcher import Searcher
+from boutiques.puller import Puller
+
+import json
+from bids_prov.utils import get_or_load
+
+DEFAULT_CONTEXT_URL = "https://raw.githubusercontent.com/cmaumet/BIDS-prov/context-type-indexing/context.json"
+TYPES = (
+    "File",
+    # "String",
+    "Number",
+    "Flag",
+)
+
+
+@get_or_load
+def get_config(agent: str) -> Mapping[str, Mapping[str, object]]:
+    """get a config by querying bosh with `agent`
```

Comment:
It would be great to explicitely mention that `bosh` is a tool to acess Boutiques documents